front/pages/w/[wId]/labs/personal_connections/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { CommandIcon, Page } from "@dust-tt/sparkle";
+import type { InferGetServerSidePropsType } from "next";
+
+import { ConversationsNavigationProvider } from "@app/components/assistant/conversation/ConversationsNavigationProvider";
+import { LabsPersonalConnectionsList } from "@app/components/data_source/LabsPersonalConnectionsList";
+import AppLayout from "@app/components/sparkle/AppLayout";
+import { getFeatureFlags } from "@app/lib/auth";
+import { withDefaultUserAuthRequirements } from "@app/lib/iam/session";
+import type { SubscriptionType, WorkspaceType } from "@app/types";
+
+export const getServerSideProps = withDefaultUserAuthRequirements<{
+  owner: WorkspaceType;
+  subscription: SubscriptionType;
+}>(async (_context, auth) => {
+  const owner = auth.workspace();
+  const subscription = auth.subscription();
+  const user = auth.getNonNullableUser();
+
+  if (!owner || !subscription || !user) {
+    return {
+      notFound: true,
+    };
+  }
+
+  const flags = await getFeatureFlags(owner);
+  if (!flags.includes("labs_personal_connections")) {
+    return {
+      notFound: true,
+    };
+  }
+
+  return {
+    props: {
+      owner,
+      subscription,
+    },
+  };
+});
+
+export default function PersonalConnections({
```

Comment:
It thought we agreed to make it something very Salesforce specific. This is to broad and will prompt our users to ask questions and believe we have broad support for that.

Please refactor to a Salesforce specific presentation🙏